storage/default__vector_store.json

```diff
@@ -0,0 +1 @@
+{ "embedding_dict": {}, "text_id_to_ref_doc_id": {}, "metadata_dict": {} }
```

Comment:
do we need to commit those json files?

Reply:
No, they were created by make format and make lint for some reason, added automatically, so I did not touch them.

Reply:
could we remove them 
I don't think they are needed

Reply:
Ok sure

Reply:
Done @hatianzhang , please check